match/match.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+# Get the userID and University name
+USER_ID = "UID3"  # TODO: connect to db
+USER_UNI = "UCLA"  # TODO: connect to db
+
+
+def main():
+    # open the file
+    with open("data.json", "r") as read_file:  # TODO: connect to db
+        data = json.load(read_file)
+
+    # provided data from the user: interests, name, [previous matches]
+    user_data = data["Universities"][USER_UNI][USER_ID]
+
+    # all users of the university
+    uni_users = data["Universities"][USER_UNI]
```

Comment:
For `uni_users` we'll have to change this to a for loop that parses the users and checks the 'university' tag. I messed this up and didn't realize that we couldn't subvert the users inside of the universities branch because at the time of parsing for displaying on the website we don't know which university they belong to or where the uid is in the db

Something like
```
for user in data['users']:
        if user['university'] == USER_UNI:
                  uni_users.append(user)
```